tools/tsp-client/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Release
 
+## 2025-04-07 - 0.18.0
+
+- Specify tsp-client specific configurations under a `@azure-tools/typespec-client-generator-cli` entry in tspconfig.yaml options. This affects the `additionalDirectories` configuration. Example entry in tspconfig.yaml:
+
+```yaml
+options:
+  "@azure-tools/typespec-client-generator-cli":
```

Comment:
For ease of use I would prefer to have this be:
```suggestion
  "tsp-client":
```

Reply:
it would be cleaner but also inconsitent with the other options, at least here we know who is using that info

Reply:
Ok, I'll leave as is for now and see if there's any push back